src/components/LayoutEditor.vue

```diff
@@ -27,15 +27,13 @@ export default {
 }
 
 function nextView(view) {
-  switch (view) {
-    case 'editor':
-      return 'code'
-    case 'code':
-      return 'props'
-    default:
-      return 'editor'
+  const viewTypes = {
+  editor: 'code',
+  code: 'props',
   }
```

Comment:
Could you rename this variable to nextViewMap? And we can also extract it out of the nextView function so the object is not created each time it is called 👍